src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -699,14 +704,14 @@ public AdsBaseObject triggerAdSlotWithMobileState(String slotName, Boolean isMob
     return this;
   }
 
-  private void triggerFMR() {
+  private void triggerFMR(String slotName) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "triggerFMR" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)